ui/packages/consul-ui/app/abilities/license.js

```diff
@@ -1,6 +1,13 @@
 import BaseAbility from './base';
+import { inject as service } from '@ember/service';
 
 export default class LicenseAbility extends BaseAbility {
   resource = 'operator';
   segmented = false;
+
+  @service('env') env;
+
+  get canRead() {
+    return this.env.var('CONSUL_NSPACES_ENABLED') && super.canRead;
```

Comment:
@jherschman For my clarification, the `CONSUL_NSPACES_ENABLED` env var is the one that we use to determine whether a customer is OSS or Ent?

Reply:
@evrowe yea that's what I have been enabling to to QA enterprise!  If you want / dont already have you can create a bookmark with this if you want an easy way to toggle it on: ` javascript:Scenario('CONSUL_NSPACES_ENABLE=1')`